nengo_dl/converter.py

```diff
@@ -1457,11 +1452,11 @@ class ConvertInput(LayerConverter):
     """Convert ``tf.keras.layers.InputLayer`` to Nengo objects."""
 
     def convert(self, node_id):
-        # if this input layer has an input obj, that means it is a passthrough
-        # (so we just return the input)
-        output = self.get_input_obj(node_id)
-
-        if output is None:
+        try:
+            # if this input layer has an input obj, that means it is a passthrough
+            # (so we just return the input)
+            output = self.get_input_obj(node_id)
+        except KeyError:
```

Comment:
Ahh, I see, this is the one place we needed that, so then we just catch the error. Works for me, as long as there's not any risk with catching some other KeyError by mistake.